e2e/tests/tests/eigen_state.rs

```diff
@@ -69,10 +68,11 @@ async fn test_eigen_state() -> Result<()> {
     // Wait for some blocks to be processed
     tokio::time::sleep(Duration::from_secs(30)).await;
 
+    // TODO: we should investigate directly querying the database instead of using metrics.
     // Check if committer has processed any blocks
     let metrics = client.get(metrics_url).send().await?.text().await?;
     assert!(
-        metrics.contains("blocks_processed_total"),
+        metrics.contains("height_of_latest_commitment"),
```

Comment:
i think this refers to the state roots posted to the L1, not the actual blobs that were posted, did i understand this correctly?

Reply:
Yes, so from my understanding, it should be the latest batch { height of blocks } posted to L1.

Reply:
i think that's an invalid metric to look for then because we're concerned with blobs posted not state root commitments :/